jmeter/src/main/java/org/eclipse/hono/jmeter/client/HonoReceiver.java

```diff
@@ -81,7 +87,8 @@ public HonoReceiver(final HonoReceiverSampler sampler) throws InterruptedExcepti
     private void connect() throws InterruptedException {
         final CountDownLatch receiverConnectLatch = new CountDownLatch(1);
         amqpNetworkClient = new HonoClientImpl(vertx, amqpNetworkConnectionFactory);
-        amqpNetworkClient.connect(getClientOptions(), connectionHandler -> {
+        amqpNetworkClient.connect(getClientOptions(Integer.parseInt(sampler.getReconnectAttempts())),
+                connectionHandler -> {
```

Comment:
Could throw a NumberFormatException.

Reply:
thats ok right? as it will result in an JMeter Test failure.

Reply:
Yes, but for a user without the code it might be better to give him a hint to the reconnect attempts field, I think.